recipes/perl-go-perl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "perl-go-perl" %}
+{% set version = "0.15" %}
+{% set sha256 = "423d26155ee85ca51ab2270cee59f4e85b193e57ac3a29aff827298c0a396b12" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: go-perl-0.15.tar.gz
+  url: https://cpan.metacpan.org/authors/id/C/CM/CMUNGALL/go-perl-0.15.tar.gz
```

Comment:
Please use the variable for the version here as well.

Reply:
Done